cmd/snap/cmd_snap_op.go

```diff
@@ -142,7 +146,19 @@ func (x *cmdInstall) Execute([]string) error {
 		return err
 	}
 
-	return wait(cli, changeID)
+	if err := wait(cli, changeID); err != nil {
```

Comment:
Should we change wait so it returns the last change, so we don't need to re-query below?